develop/items/custom-enchantment-effects.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Custom Enchantment Effects
+description: Learn how to create your enchantment effects.
+authors:
+  - krizh-p
+---
+
+# Custom Enchantments {#custom-enchantments}
+
+Starting from version 1.21, custom enchantments in Minecraft use a "data-driven" approach. This makes it easier to add simple enchantments, like increasing attack damage, but more challenging to create complex ones. The process involves breaking down enchantments into _effect components_.
+
+An effect component contains the code that defines the special effects of an enchantment. Minecraft supports various default effects, such as item damage, knockback, and experience. 
+
+However, this guide focuses on creating custom enchantment effects that aren't supported by default.
+
+**Before continuing, please check if the default Minecraft effects can work for your needs by visiting [the Minecraft Wiki's Enchantment Effect Components page](https://minecraft.wiki/w/Enchantment_definition#Effect_components)**. The rest of the guide assumes you understand how to configure "simple" data-driven enchantments.
```

Comment:
```suggestion
An effect component contains the code that defines the special effects of an enchantment. Minecraft supports various default effects, such as item damage, knockback, and experience. 

::: tip
Be sure to check if the default Minecraft effects satisfy your needs by visiting [the Minecraft Wiki's Enchantment Effect Components page](https://minecraft.wiki/w/Enchantment_definition#Effect_components). This guide assumes you understand how to configure "simple" data-driven enchantments and focuses on creating custom enchantment effects that aren't supported by default.
:::
```

Reply:
Not sure why it isn't letting me commit the suggestion. @krizh-p Would you be able to make these changes manually?